glotaran/cli/commands/pluginlist.py

```diff
@@ -11,11 +11,11 @@ def plugin_list_cmd():
     output = """
     Installed Glotaran Plugins:
 
-    Models:
+    Megacomplex Models:
     """
```

Comment:
We should use `dedent` on this (`from textwrap import dedent`).
The current output looks like this:
```console
$ glotaran pluginlist

    Installed Glotaran Plugins:

    Megacomplex Models:

    * baseline
    * coherent-artifact
    * decay
    * spectral

Data file Formats

    * .ascii
    * .nc
    * .sdt

Project file Formats

    * .csv
    * .folder
    * .legacy
    * .yaml
    * .yml
    * .yml_str
```